cmd/cilium/main.go

```diff
@@ -16,12 +16,19 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"os"
 
 	"github.com/cilium/cilium-cli/internal/cli/cmd"
+
+	gops "github.com/google/gops/agent"
 )
 
 func main() {
+	if err := gops.Listen(gops.Options{}); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Do we want to run gops unconditionally? It seems to me that it should only be run in debug mode or something, no?

Reply:
Not sure why we could not run gops unconditionally. I believe we do that in cilium-agent as well.

Reply:
We do but we bind to a specific port. In any case, I think it'd be better to just log a warning rather than fatal when gops cannot be started.

Reply:
Changed `log.Fatal()` to `log.Printf()` & rebased.